eagleproject/core/views.py

```diff
@@ -881,6 +881,40 @@ def subscribe(request):
     else:
         return render(request, 'subscription.html', {'form': SubscriberForm()})
 
+@csrf_exempt
```

Comment:
Not having the csrf opens up this route to being open to an attacker flooding it with requests and potentially filling up the DB with fake/bad data.

If csrf for some reason does not work let's at least setup some rate limiting. This can be done in CloudFlare - let me know and I can help.

Also see https://github.com/oplabs/security/issues/21